contrib/parseq-retry/src/main/java/com/linkedin/parseq/retry/monitor/LogEventsWithJava.java

```diff
@@ -0,0 +1,84 @@
+package com.linkedin.parseq.retry.monitor;
+
+import com.linkedin.parseq.function.Try;
+
+import java.util.Optional;
+import java.util.function.Function;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+
+/**
+ * An event monitor that formats and logs events using the `java.util.logging` framework.
+ *
+ * @param <T> Type of a task result, used for strongly typed processing of outcomes.
+ *
+ * @author Oleg Anashkin (oleg.anashkin@gmail.com)
+ */
+public class LogEventsWithJava<T> extends LogEvents<T, Level> {
```

Comment:
This class doesn't seem necessary since we already have implementation against SLF4J API.